src/hammer-vlsi/defaults.yml

```diff
@@ -345,7 +345,7 @@ vlsi.inputs:
   # Valid options:
   # cpf - Use the common power format commonly used in Cadence tools
   # upf - Use the universal power format, IEEE 1801-2015
-  power_spec_type: null
+  power_spec_type: upf
```

Comment:
I think we should use cpf as a default given that has the most supported

Reply:
On second thought since the default is `power_spec_mode: empty`, if a user sets `power_spec_mode` to manual without specifying a value, shouldn't it trigger an error?